src/main/java/com/github/tomakehurst/wiremock/common/xml/Xml.java

```diff
@@ -99,13 +99,9 @@ public static String prettyPrint(Document doc) {
     }
 
     private static TransformerFactory createTransformerFactory() {
-        try {
-            TransformerFactory transformerFactory = (TransformerFactory) Class.forName("com.sun.org.apache.xalan.internal.xsltc.trax.TransformerFactoryImpl").getDeclaredConstructor().newInstance();
-            transformerFactory.setAttribute("indent-number", 2);
-            return transformerFactory;
-        } catch (Exception e) {
-            return TransformerFactory.newInstance();
-        }
+        TransformerFactory transformerFactory = TransformerFactory.newInstance();
```

Comment:
TransformerFactory.newInstance has a fallback to `com.sun.org.apache.xalan.internal.xsltc.trax.TransformerFactoryImpl`, so it seemed to me like we can call this directly.